testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/CLIEmbedServerTestCase.java

```diff
@@ -795,9 +798,9 @@ private void testPathDir(final String propName, final String value) throws IOExc
             for(String prop : SERVER_PROPS) {
                 if (prop.equals(propName)) {
                     assertPath(propName, ROOT + File.separator + newStandalone + File.separator + value);
-                } else {
-                    // just make sure the unchanged property has the default basedir
-                    //assertTrue(WildFlySecurityManager.getPropertyPrivileged(prop, "").indexOf(currBaseDir) != -1);
+//                } else {
```

Comment:
The assert was already commented out. I just commented out the whole else block instead of just that line to stop my IDE warning me about an empty else block.

If the assert should be uncommented, I can stick in a TODO to get that done. I don't want to uncomment it as part of this though.